internal/app/connectconformance/testsuites/connect_only.yaml

```diff
@@ -0,0 +1,101 @@
+    requestMessages:
+    - "@type": type.googleapis.com/connectrpc.conformance.v1alpha1.UnaryRequest
+      responseDefinition:
+        responseHeaders:
+        - name: x-custom-header
+          value: ["foo","bar","baz"]
+        error:
+          code: 13
+          message: "unary failed"
+          details:
+            - "@type": "connectrpc.conformance.v1alpha1.Header"
+              name: "test error detail name"
+              value:
+                - "test error detail value"
+        responseTrailers:
+        - name: x-custom-trailer
+          value: ["bing","quux"]
+  expectedResponse:
+    responseHeaders:
+    - name: x-custom-header
+      value: ["foo","bar","baz"]
+    error:
+      code: 13
+      message: "unary failed"
+      details:
+        - "@type": "connectrpc.conformance.v1alpha1.Header"
+          name: "test error detail name"
+          value:
+            - "test error detail value"
+    responseTrailers:
+    - name: X-Custom-Trailer
+      value: ["bing","quux"]
+- request:
+    testName: client stream error
+    service: connectrpc.conformance.v1alpha1.ConformanceService
+    method: ClientStream
+    streamType: STREAM_TYPE_CLIENT_STREAM
+    requestHeaders:
+    - name: X-Conformance-Test
+      value: ["Value1","Value2"]
+    requestMessages:
+    - "@type": type.googleapis.com/connectrpc.conformance.v1alpha1.ClientStreamRequest
+      responseDefinition:
+        responseHeaders:
+        - name: x-custom-header
+          value: ["foo","bar","baz"]
+        error:
+          code: 13
+          message: "client stream failed"
+          details:
+            - "@type": "connectrpc.conformance.v1alpha1.Header"
+              name: "test error detail name"
+              value:
+                - "test error detail value"
+        responseTrailers:
+        - name: x-custom-trailer
+          value: ["bing","quux"]
+    - "@type": type.googleapis.com/connectrpc.conformance.v1alpha1.ClientStreamRequest
+      requestData: "dGVzdCByZXNwb25zZQ=="
+  expectedResponse:
+    responseHeaders:
+    - name: X-Custom-Header
+      value: ["foo","bar","baz"]
+    error:
+      code: 13
+      message: "client stream failed"
+      details:
+        - "@type": "connectrpc.conformance.v1alpha1.Header"
+          name: "test error detail name"
+          value:
+            - "test error detail value"
+    responseTrailers:
+    - name: X-Custom-Trailer
+      value: ["bing","quux"]
+- request:
+    testName: unimplemented
+    service: connectrpc.conformance.v1alpha1.ConformanceService
+    method: Unimplemented
+    streamType: STREAM_TYPE_UNARY
+    requestHeaders:
+    - name: X-Conformance-Test
+      value: ["Value1","Value2"]
+    requestMessages:
+    - "@type": type.googleapis.com/connectrpc.conformance.v1alpha1.UnimplementedRequest
+  expectedResponse:
+    error:
+      code: 12
+      message: "connectrpc.conformance.v1alpha1.ConformanceService.Unimplemented is not implemented"
```

Comment:
@jhump We probably don't want to validate the actual error message text here since an implementation can put whatever they want right? I think we'd only want to validate the code, but I don't think we have a way to do that.